lametro/events.py

```diff
@@ -206,6 +205,17 @@ def scrape(self, window=None) :
                         note = "Agenda number, {}".format(item["EventItemAgendaNumber"])
                         agenda_item['notes'].append(note)
 
+                    # The EventItemAgendaSequence provides 
+                    # the line number of the Legistar agenda grid.
+                    agenda_item['extras']['item_agenda_sequence'] = item['EventItemAgendaSequence']
+
+                # Historically, the Legistar system has duplicated the EventItemAgendaSequence,
+                # resulting in data inaccuracies. The scrape should fail in such cases, until Metro
+                # cleans the data.
+                item_agenda_sequences = [item['extras']['item_agenda_sequence'] for item in e.agenda]
+                if len(item_agenda_sequences) != len(set(item_agenda_sequences)):
+                    raise ValueError('Agenda for Event {} has duplicate agenda items on the Legistar grid. Contact Metro, and ask them to remove the duplicate EventItemAgendaSequence.'.format(e.pupa_id))
```

Comment:
could we include the identifier (or something that would allow metro to locate the duplicates) in this message? that way, we can act on them without having to do any further sleuthing.

Reply:
yup, yup, yup